Formula/itstool.rb

```diff
@@ -19,14 +20,12 @@ class Itstool < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libxml2"
-  depends_on "python@3.9"
+  depends_on "python@3.10"
 
   def install
-    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
-    ENV.append_path "PYTHONPATH", "#{Formula["libxml2"].opt_lib}/python#{xy}/site-packages"
+    ENV.append_path "PYTHONPATH", Formula["libxml2"].opt_prefix/Language::Python.site_packages("python3")
```

Comment:
Is this still necessary, since the `site-packages` directory is in `prefix` (and not, e.g., `libexec`)?

Reply:
`libxml2` is keg-only on macOS, so won't get picked up by default.

Reply:
Ah right. I wonder if we could make it not keg-only...